src/main/java/jarvis/Jarvis.java

```diff
@@ -0,0 +1,56 @@
+package jarvis;
+
+import java.util.Scanner;
+
+import jarvis.command.Command;
+import jarvis.parser.Parser;
+import jarvis.storage.Storage;
+import jarvis.task.TaskList;
+import jarvis.ui.Ui;
+
+/**
+ * Jarvis --- Task manager.
+ */
+public class Jarvis {
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Constructor.
+     */
+    public Jarvis() {
+        this.storage = new Storage(System.getProperty("user.dir") + "/data/tasks.txt");
+        this.ui = new Ui();
+        try {
+            tasks = new TaskList(storage.loadTasks());
+        } catch (JarvisException e) {
+            this.ui.printMessage("Error loading tasks.");
+        }
+    }
+
+    /**
+     *
+     */
+    public void run() {
+        this.ui.greet();
+
+        Scanner sc = new Scanner(System.in);
+        boolean terminated = false;
```

Comment:
Got your idea that the Boolean tracks whether the chat is still ongoing.
Should the Boolean name be a verb instead?
```suggestion
        boolean isTerminated = false;
````